staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/strategy.go

```diff
@@ -72,6 +73,23 @@ func (a customResourceStrategy) NamespaceScoped() bool {
 	return a.namespaceScoped
 }
 
+// GetResetFields returns the set of fields that get reset by the strategy
+// and should not be modified by the user.
+func (a customResourceStrategy) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set {
+	fields := map[fieldpath.APIVersion]*fieldpath.Set{}
+
+	if a.status != nil {
```

Comment:
same here

> i'm not even sure if this is correct.
> we are in the customresource strategy, shouldn't the version be the respective CR version and not apiextensions.k8s.io/v1?

but, we only add the resetFields if status is a subresource, that plays along with https://github.com/kubernetes/kubernetes/pull/92809/files/28a5e20d8d489c4845c1ccc55e8223aca7476a6d..d98641976458e0e4ead9eac950e4d5fdd4714d6d#diff-34aedeefa7784abbe7531bae68833ba6R95